packages/docutils/base-mkdocs.yml

```diff
@@ -14,6 +14,8 @@ markdown_extensions:
       emoji_index: !!python/name:materialx.emoji.twemoji
       emoji_generator: !!python/name:materialx.emoji.to_svg
   - footnotes
+  - markdown_include.include:
+      base_path: ../../typedoc-docs/commands
```

Comment:
will this work when used by an extension? we don't want extension authors to have to recreate this directory structure right?

Reply:
not as written, no.  will be separate PR